src/applications/vaos/services/appointment/transformers.js

```diff
@@ -598,6 +598,7 @@ export function transformPendingAppointments(requests) {
       resourceType: 'Appointment',
       id: `var${appt.id}`,
       status: getRequestStatus(appt, isExpressCare),
+      created: moment(appt.createdDate, 'MM/DD/YYYY HH:mm:SS').format(),
```

Comment:
Specs call for list to be sorted by most recently created date descending.  Added this to the transformer since we weren't capturing that field before.